zinnia/signals.py

```diff
@@ -29,8 +29,10 @@
 PINGBACK_WF_COUNT_PINGBACKS = 'zinnia.pingback.was_flagged.count_pingbacks'
 TRACKBACK_WF_COUNT_TRACKBACKS = 'zinnia.trackback.was_flagged.count_trackbacks'
 
-pingback_was_posted = Signal(providing_args=['pingback', 'entry'])
-trackback_was_posted = Signal(providing_args=['trackback', 'entry'])
+# providing_args=['pingback', 'entry']
+pingback_was_posted = Signal()
+# providing_args=['trackback', 'entry']
+trackback_was_posted = Signal()
```

Comment:
Apparently it stopped being supported

Django (2.2)[https://docs.djangoproject.com/en/2.2/topics/signals/#defining-signals] vs (3.2)[https://docs.djangoproject.com/en/3.2/topics/signals/#defining-signals]